### PR TITLE
App Modelio.app v3.4.1

### PR DESCRIPTION
**Pull request by michaelPf85**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by michaelPf85** [comment]
I think I made the pull request on the wrong commit, before I fixed 2 style issues :/ Gonna wait for Travis and check this.

**Comment by michaelPf85** [comment]
After fixing the style issues, Travis fails with absolutely no error message ! It looks like the XCode 7.2 build is failing for all pull requests event though no offense is detected.

File: Casks/modelio.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask 'modelio' do
+  version '3.4.1'
+  sha256 'ad31345e88b287cce90b21cf35ca801bc24befcfb4bd2dfc25a3781e4fc415fb'
+
+  url 'https://www.modelio.org/modelio-download-archive/doc_download/118-modelio-341-macos-x.html'
+  name 'Modelio'
+  homepage 'https://www.modelio.org/'
+  # Core is GPL and Module Runtime is APL (https://www.modelio.org/about-modelio/license.html)
```
Comment (vitorgalvao) [inline Casks/modelio.rb]: Please remove the commend.

File: Casks/modelio.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  license :oss
+
+  app 'Modelio 3.4/modelio.app'
```
Comment (vitorgalvao) [inline Casks/modelio.rb]: `app "Modelio #{version}/modelio.app"`.

**Comment by vitorgalvao** [comment]
The failure is on Travis. It seems to be choking, these last few days.

File: Casks/modelio.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+cask 'modelio' do
+  version '3.4.1'
```
Comment (vitorgalvao) [inline Casks/modelio.rb]: `version '3.4.1,118'`

File: Casks/modelio.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  sha256 'ad31345e88b287cce90b21cf35ca801bc24befcfb4bd2dfc25a3781e4fc415fb'
+
+  url 'https://www.modelio.org/modelio-download-archive/doc_download/118-modelio-341-macos-x.html'
```
Comment (vitorgalvao) [inline Casks/modelio.rb]: Are you sure this works with homebrew-cask?

Comment (vitorgalvao) [inline Casks/modelio.rb]: Use [`before_comma` and `after_comma`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/version.md#version-methods).

Comment (michaelPf85) [inline Casks/modelio.rb]: I did test it on my MacBook with homebrew-cask, why ?
It does download an "html" file which is indeed a zip file... Strange way to handle download link from their part. Cask unzips it properly.

**Comment by michaelPf85** [comment]
Travis seems to be failing on all Xcode 7.2 builds from what I've seen from several other PRs :/ I brought the subject on Gitter , no feedback there yet.

**Comment by michaelPf85** [comment]
It's my first PR, I don't know if I'm doing things properly.

I amended my stuff to keep everything in a single commit and it now shows your comments as "outdated", but I think I included them all.

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in c911261b6119cda548e49d39b55b83f9e425656c. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by adidalal** [comment]
@michaelPf85 I actually am not sure of linking just the app will work, but feel free to brew update and try it - if not, we'll have to use the `suite` artifact.

**Comment by michaelPf85** [comment]
Thanks for the improvements and the merge ! I tried the "no_dots" trick but it didn't work, I must have miss-typed something.

The "app" linking works perfectly, I triple-checked and walked around the app after a brew update / install. It looks like a "false" app that just run a small executable which itself seems to spawn the Java application, so I might be lucky with this one.

**Pull request closed without merging**